keras_contrib/optimizers/padam.py

```diff
@@ -0,0 +1,94 @@
+from keras import backend as K
+from keras.optimizers import Optimizer
+
+
+class Padam(Optimizer):
+    def __init__(self, lr=1e-1, beta_1=0.9, beta_2=0.999,
+                 epsilon=1e-8, decay=0., amsgrad=False, partial=1. / 8., **kwargs):
+        """ Partially adaptive momentum estimation optimizer.
+
+        # Arguments
+            lr: float >= 0. Learning rate.
+            beta_1: float, 0 < beta < 1. Generally close to 1.
+            beta_2: float, 0 < beta < 1. Generally close to 1.
+            epsilon: float >= 0. Fuzz factor. If `None`, defaults to `K.epsilon()`.
+            decay: float >= 0. Learning rate decay over each update.
+            amsgrad: boolean. Whether to apply the AMSGrad variant of this
+                algorithm from the paper "On the Convergence of Adam and
+                Beyond".
+            partial: float >=0. Parameter controlling partial momentum adaption.
```

Comment:
Document here that it should be >= 0 and <= 0.5.  Also, add a line that states what the two extreme ranges exhibit - 0.0 exhibiting SGD behavior and 0.5 exhibiting AMSGrad behavior.